azure-pipelines.yml

```
@@ -76,7 +76,7 @@ jobs:
         inputs:
           packageType: 'sdk'
           version: '$(LATEST_NET_VERSION)'
-          includePreviewVersions: false
+          includePreviewVersions: true
```

Reminder to set this to false before merging
```suggestion
          includePreviewVersions: false
```